metapackages/auto-instrumentations-node/src/register.ts

```diff
@@ -25,8 +25,17 @@ diag.setLogger(
   opentelemetry.core.getEnv().OTEL_LOG_LEVEL
 );
 
+const disabledAutoinstrumentations = (
+  process.env.OTEL_DISABLED_AUTOINSTRUMENTATIONS ?? ''
```

Comment:
This environment variable is not speced. There is the [`OTEL_SDK_DISABLED`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/sdk-environment-variables.md#general-sdk-configuration) environment variable from spec which I think might be what you are looking for. 

However, I think OTEL should be disabled in the SDK level and not in the instrumentations level. Or are you interested in installing an SDK and only having auto instrumentations disabled while allowing manual instrumentation via trace/metrics/logs providers?

What distro are you currently using to instrument your code?